samples/dashboards/sync/highcharts-highlight-affected-series/demo.js

```diff
@@ -0,0 +1,95 @@
+const board = Dashboards.board('container', {
+    dataPool: {
+        connectors: [{
+            id: 'csv',
+            type: 'CSV',
+            options: {
+                csv: document.getElementById('csv').innerText
+            }
+        }]
+    },
+    gui: {
+        layouts: [{
+            id: 'layout-1',
+            rows: [{
+                cells: [{
+                    id: 'dashboard-col-0'
+                }, {
+                    id: 'dashboard-col-1'
+                }]
+            }]
+        }]
+    },
+    components: [{
+        renderTo: 'dashboard-col-0',
+        type: 'Highcharts',
+        connector: [{
+            id: 'csv',
+            columnAssignment: [{
+                seriesId: 'Y2020',
+                data: ['Name', 'Y2020']
+            }, {
+                seriesId: 'Y2021',
+                data: ['Name', 'Y2021']
+            }, {
+                seriesId: 'Y2022',
+                data: ['Name', 'Y2022']
+            }]
+        }],
+        sync: {
+            highlight: true
+        },
+        chartOptions: {
+            xAxis: {
+                type: 'category'
+            },
+            series: [{
+                type: 'column',
+                name: '2020',
+                id: 'Y2020'
+            }, {
+                type: 'column',
+                name: '2021',
+                id: 'Y2021'
+            }, {
+                type: 'column',
+                name: '2022',
+                id: 'Y2022'
+            }]
+        }
+    }, {
+        renderTo: 'dashboard-col-1',
+        type: 'DataGrid',
+        title: {
+            text: 'csv 1'
+        },
+        connector: {
+            id: 'csv'
+        },
+        sync: {
+            highlight: true
+        }
+    }]
+});
+
+
+/**
+ * Custom code
+ **/
+
+const optionsCbx = document.querySelectorAll('.option');
```

Comment:
Can we replace `Cbx` with something more meaningful?

Reply:
Oh, right, I copied it from my old demo with checkboxes and didn't update the name even though there are radio buttons here... 🙈